graphql_api/helpers/mutation.py

```diff
@@ -57,5 +58,20 @@ def authenticated_resolver(instance, info, *args, **kwargs):
     return authenticated_resolver
 
 
+def require_part_of_org(resolver):
```

Comment:
is it easy to add a UT for this guy?

Reply:
this function should be fully covered with the test_owner.py tests